test/function/samples/plugin-error-transform-pos/_config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('node:path');
+
+module.exports = defineTest({
+	description: '`this.error(...)` accepts number as second parameter (#5044)',
+	options: {
+		plugins: [
+			{
+				name: 'plugin1',
+				transform(code) {
+					return code + 'bar';
+				}
+			},
+			{
+				name: 'plugin2',
+				transform() {
+					this.error(new Error('error'), 26);
+				}
+			}
+		]
+	},
+	error: {
+		code: 'PLUGIN_ERROR',
+		message: 'error',
+		plugin: 'plugin2',
+		hook: 'transform',
+		id: path.join(__dirname, 'main.js'),
+		watchFiles: [path.join(__dirname, 'main.js')],
+		frame: "11: export const foo = 'foo'bar\n                             ^",
```

Comment:
I'm not sure why the test fails when `frame` is `1: export const foo = 'foo'bar\n                             ^`.

Reply:
The reason is that `deindent` function is slicing the string from index 1 (see https://github.com/rollup/rollup/blob/833314c1621420e031340c1894e7863e062f7761/test/utils.js#L104). You can use "Template literals" like this
```js
frame: `
1: export const foo = 'foo'bar
                                               ^
`,
```

Reply:
Ah, I see. Thank you!